src/display/annotation_layer.js

```diff
@@ -620,7 +621,8 @@ class ChoiceWidgetAnnotationElement extends WidgetAnnotationElement {
 
     const selectElement = document.createElement("select");
     selectElement.disabled = this.data.readOnly;
-
+    selectElement.setAttribute("name", this.data.fieldName);
```

Comment:
`name` should be a standard attribute, so here and above using `element.name = this.data.fieldName` should also work and is more consistent with the other lines. (In fact, we should always use direct attribute assignment except for non-standard attributes, so if there are other occurrences in this file, feel free to change them in a separate commit.)

Reply:
Hi @timvandermeij I haved replaced setAttribute with direct assignment and it works.